src/correctColor/__tests__/correctColor.test.ts

```diff
@@ -136,7 +136,7 @@ describe('correctColor', () => {
 
     const result = correctColor(image, measuredColors, referenceColors);
 
-    // all colors are nearly perfectly corrected, expect the gray
+    // All colors are nearly perfectly corrected, expect the gray.
```

Comment:
If "expect" is what you intended to write, I don't understand because the entire result is verified.

Reply:
The result is verified indeed. I just left this comment because I found it interesting that all the colors were corrected in the result ( no more offset in the squares), excepted in the grey square, where some variation is still visible.

Reply:
Then I think my suggestion was correct?
![image](https://user-images.githubusercontent.com/2352663/174813935-fd74ec09-d6f4-4e57-b187-adf63c105cdb.png)

Reply:
Woopsie, my bad